sdk/cognitiveservices/ci.yml

```diff
@@ -0,0 +1,23 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - sdk/cognitiveservices/
+
+pr:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - sdk/cognitiveservices/
+
+jobs:
+- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+  parameters:
+    BuildTargettingString: azure-cognitiveservices-formrecognizer,azure-cognitiveservices-knowledge-nspkg,azure-cognitiveservices-language-nspkg,azure-cognitiveservices-nspkg,azure-cognitiveservices-search-nspkg,azure-cognitiveservices-vision-nspkg
```

Comment:
Is `BuildTargetingString` auto-generated based on file structure? We've been churning _super_ hard on the repo restructure, so I expect a bunch of these to change in the near future.

Reply:
Even though the file says that it is auto-generated - it isn't. So these are hand written. This file is becoming something of a customization configuration per pipeline where values that control behavior deeper in the templates are passed in.

We might remove the auto generated notice at the top of the file soon. The upshot is that these options are hand-written. If they aren't right we can change them.

Reply:
Ok! Sounds good.